recipe/meta.yaml

```diff
@@ -11,29 +11,36 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
+  skip: True  # [py<37]
 
 requirements:
   host:
     - pip
-    - python >=3.7
-    - poetry
+    - python
+    - poetry-core 1.5.1
     - poetry-dynamic-versioning
```

Comment:
pinning ?

Reply:
Let's assume we need any version and it's a Python package so we can drop exact pinning